### PR TITLE
Move Successor space between Coalition space and Elenchus base

### PR DESCRIPTION
**Pull request by DarcyManoel**
Successor space remains out of jump range of both areas it lies between. This is to ensure consistency between system placement and the galaxy image, and to keep any region additions to the Milky Way fit within the Milky Way.

Old
image

New
image

### CODE REVIEW

**Comment by DarcyManoel** [comment]
I would note that wormholes are reaching a point where there can be considered too many, however I don't see a suitable alternative considering the lore behind this species.

**Comment by Daeridanii1** [comment]
This is definitely an improvement since things have been out of place since the galaxy sprite rework. To give you a sense of why they were there, the original placement of the systems (on galaxy sprite 1.0) was motivated by three main desires:

1. Keeping them out of the way of other developments, and keeping the Successors isolated in general
2. Having a gameplay separation between Successor and Predecessor systems (via wormhole, jump drive, etc.)
3. Having Predecessor systems at some point contiguous with human space for translation reasons

I am concerned about moving them too close to the Coalition, partially because Arachi has stated that they ["want 14 Pole to be isolated"](https://discord.com/channels/251118043411775489/266345072554016768/845126588705669120) and I'm not sure if a placement like this infringes upon that.

My original plan for moving them post-galaxy sprite rework was slightly right of their present position on the map into that area of denser stars, which works well with the written lore that their present territory is in a star cluster on the far galactic rim.

That being said, I do like this placement because it shortens the distance of the wormhole link between their modern territory and that of the Predecessors, which is appreciable for the aesthetics of not having a purple line across half the map. Also because in conjunction with lore given to the player, the closer proximity supports suggesting that at one time the territories could have been contiguous, and general closeness just feels better for two areas that are supposed to have once been part of one political unit.

So in short; I like this change, and it fixes the problems it needs to, but it invites a larger discussion about where the Successors should be placed with regards to (at the very least) the following questions:

* What sort of interactions did the Predecessors have with other non-human species ca. 1200-more years prior to the present?
* What sort of disconnection should there be between Successor and Predecessor systems?
* What shape and placement is most appropriate for the Predecessor systems that maintains their contiguity with human space?

**Comment by DarcyManoel** [comment]
> I am concerned about moving them too close to the Coalition, partially because Arachi has stated that they ["want 14 Pole to be isolated"](https://discord.com/channels/251118043411775489/266345072554016768/845126588705669120) and I'm not sure if a placement like this infringes upon that.

This change leave 14 Pole isolated, it's not within Jump Range and thus not even visible as an empty system from it.

> What sort of interactions did the Predecessors have with other non-human species ca. 1200-more years prior to the present?

I imagine Successors/Predecessors never developed Jump Drives and so never had any meaningful relationships with Coalition or Hai. The Quarg likely would have made attempts at contact, but I don't think that's covered in lore so that can go either way.

> What shape and placement is most appropriate for the Predecessor systems that maintains their contiguity with human space?

I feel like the current shape and the new placement is appropriate, it lends itself to the idea that all of the added systems were at some point connected. The placement of the abandoned systems lends itself to contiguity with Human space, all the added systems feel natural and fit within the game.

**Comment by Daeridanii1** [comment]
Fair enough. This fixes the immediate problem of being misaligned with the sprite, so I'll merge it, though I expect that there will still be some changes to the system layout and potentially placement going forward.
